### DISCUSSION TITLE
[Feature] Share people/set of people as dynamic album

### DISCUSSION THREAD

**Discussion by PeterBasista** [immich-app/immich#3498] [Feature Request]
### The feature

With the arrival of the new human recognition feature, there is also a rather good opportunity for a few features:
- Assign a model-recognized person to the logged-in person (something similar to the iPhone where you can assign a contact to a recognized person and then dynamic memory albums are created, but your name is there as "you")
- the ability to share a recognised person/people with link. Thus the possibility to share a person like an album or to create dynamically an album where multiple multiple persons will be assigned (you select the persons and name the album - this album will be dynamically updated based on new photos of the persons) this will give you option to share photos with my friends only with friends and do not share family photo with them without creating new album/s. On people page you just click ln person and click “share” and it will give you the same options as sharing the album. Or create new album and instead of choosing photos you choose recognized and named people)
- Click on an album will aldo give the circles of the people in the album with name

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by vulcanjedi** [comment]
[https://github.com/immich-app/immich/discussions/3306](url)

**Comment by shivardev** [comment]
Looking for this

**Comment by alangrainger** [comment]
I made a quick Docker image which will automatically keep an album or shared album up-to-date with a person/face or set of people:

https://github.com/alangrainger/immich-person-to-album/

I'm sure this will arrive in core Immich at some point, but this should help in the interim.
